app/forms/common/YesNoAnswer.scala

```diff
@@ -44,3 +47,13 @@ object YesNoAnswer {
   def form(fieldName: String = "yesNo", errorKey: String = "error.yesNo.required"): Form[YesNoAnswer] =
     Form(mapping(fieldName, errorKey))
 }
+
+object IsAuthorisationRequired extends ClearanceModifierPage {
+  override def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] =
+    Seq(TariffContentKey(s"tariff.declaration.isAuthorisationRequired.${defineJourneySpecificKey(decType)}"))
+}
+
+object PreviousDocumentsRemove extends DeclarationPage {
+  override def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] =
+    Seq(TariffContentKey("tariff.declaration.previousDocuments.remove.clearance"))
+}
```

Comment:
You can find appropriate objects in files `forms.declaration.Document` and `forms.declaration.DeclarationHolder`.

Reply:
Thanks, removed these and used the existing objects.